src/main/webapp/app/shared/sidebar/sidebar-accordion/sidebar-accordion.component.ts

```diff
@@ -39,6 +41,7 @@ export class SidebarAccordionComponent implements OnChanges, OnInit {
         } else {
             this.setStoredCollapseState();
         }
+        this.calculateTotalUnreadMessages();
```

Comment:
isn't it kind of redundant, when you do this call in ngOnInit and ngOnChanges? Wouldn't it be enough to just do it in ngOnChanges?